insights/specs/default.py

```diff
@@ -89,6 +89,7 @@ def tomcat_base(broker):
     ceph_config_show = foreach_execute(ceph_socket_files, "/usr/bin/ceph daemon %s config show")
     ceph_df_detail = simple_command("/usr/bin/ceph df detail -f json-pretty")
     ceph_health_detail = simple_command("/usr/bin/ceph health detail -f json-pretty")
+    ceph_insights = simple_command("/usr/bin/ceph insights")
```

Comment:
Let's wait until #1455 gets merged before adding the datasource implementation. We can leave the `RegistryPoint`, but we want to protect this command from running on every machine. It'll need to depend on a separate component that figures out whether it should run or not so we we don't stampede the cluster.

For development I can send you an example of how to use a datasource implementation that hooks into a `RegistryPoint` without needing to be in core.

Reply:
Sounds great, I'll keep an eye on #1455. I'm going to dig into the instructions you sent for running end-to-end examples on Friday or Monday, so I could look at both at the same time.